src/main/java/one/util/streamex/StreamEx.java

```diff
@@ -299,6 +315,38 @@
         return new StreamEx<>(new PairSpliterator.PSOfRef<>(lastMapper, notLastMapper, spliterator(), false), context);
     }
 
+    /**
+     * Returns a stream where only elements that are instances of the provided {@code clazz} are transformed using
+     * {@code mapper} and other elements are left unmodified.
+     *
+     * <p>
+     *
+     * Examples:
+     * <p>
+     *   List[Object] objs = StreamEx.of(1, "a") <br>
```

Comment:
Code samples in JavaDoc should we wrapped with `<pre>{@code ...sample code...}</pre>`. There you can use `List<Object>` without problems and don't need to add `<br>`. Also they will be formatted like code.